src/lib/prefixes.ts

```diff
@@ -9,6 +9,7 @@ export default {
   dcat: 'http://www.w3.org/ns/dcat#',
   dcterms: 'http://purl.org/dc/terms/',
   dqv: 'http://www.w3.org/ns/dqv#',
+  dtype: 'http://www.linkedmodel.org/schema/dtype',
```

Comment:
@ktk Shouldn't this be `dtype: 'http://www.linkedmodel.org/schema/dtype#'` instead, and same with `vaem:` below?